modules/ROOT/pages/design-view-source-from-exchange.adoc

```diff
@@ -0,0 +1,24 @@
+= View the Source Project for Assets Published to Exchange
+
+After publishing an asset to Anypoint Exchange, you can view the source files in the API Designer. Exchange provides a direct link to the API Designer project containing the source files.
+
+== Procedure
+
+. Open the API specification or fragment in Anypoint Exchange.
+. Open the pop-up menu for the asset version whose source you want to view.
+. Select *View code in API Designer*
++
+API Designer opens the project in read-only view.
+
+. To edit the files in the project, click *Edit spec*.
++
+This option is only available if you have edit permissions on the project and the project exists in API Designer.
+
```

Comment:
It is also important to add documentation on what happens when the Api Spec has been modified after it was published or if the view code is requested from a version that is not the last one. Api Designer will always show the latest content edited.  Then the user will get this message to be warned about new changes made to the Spec:
![image](https://user-images.githubusercontent.com/22245749/96884519-f92b3200-1457-11eb-9574-ec9609bc44e3.png)

Reply:
@carobazan be aware that this will be different, here's the Jira issue:
https://www.mulesoft.org/jira/browse/DESIGNER-6290?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel

It should be in qax this afternoon.

Reply:
@tomasmateus Can you ping me when it's available in QAX? Thanks!

Reply:
@sleslie23 it's in qax now!
Also, the PRD is now updated with every mockup.
https://docs.google.com/document/d/1nbCFV3a66-wcJnroW5D0gTZpbQCs0uEvhZM1D7rw7DM/edit#

Reply:
Updated the content as requested.